### PR TITLE
Adding required perm

### LINKED ISSUE #2293: Unclear on the minimum  role of the person that can create Custom app launcher tiles
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9488caf7-b791-3002-8749-7fdc4b2ec8b3
* Version Independent ID: 08e61eb0-7bcd-f449-46ef-17e209138a8e
* Content: [Add custom tiles to the app launcher - Microsoft 365 admin](https://docs.microsoft.com/en-us/microsoft-365/admin/manage/customize-the-app-launcher?view=o365-worldwide)
* Content Source: [microsoft-365/admin/manage/customize-the-app-launcher.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/manage/customize-the-app-launcher.md)
* Service: **o365-administration**
* GitHub Login: @twernermsft
* Microsoft Alias: **twerner**

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2293

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/manage/customize-the-app-launcher.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ You can add your own custom tiles to the app launcher that point to SharePoint s
   
 ## Add a custom tile to the app launcher
 
-1. In the admin center, go to the **Settings** > **Org Settings** and choose **Organization profile** tab.
+1. Log into the admin center as a Global Administrator, go to the **Settings** > **Org Settings** and choose **Organization profile** tab.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/customize-the-app-launcher.md]:
```suggestion
1. Sign in to the admin center as a Global Administrator, go to **Settings** > **Org Settings**, and choose the **Organization profile** tab.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@twernermsft Please merge this PR. Thanks.

**Comment by yogkumgit** [comment]
@twernermsft Gentle reminder. Please merge this PR. Thanks.

**Pull request merged**